context/app/static/js/components/files/Files/Files.jsx

```diff
@@ -40,7 +42,12 @@ function Files(props) {
       <SectionContainer id="files">
         <SectionHeader>Files</SectionHeader>
         {files && <FileBrowser files={files} />}
-        <GlobusLink uuid={uuid} hubmap_id={hubmap_id} />
+        <MarginTopDiv>
+          <DetailSectionPaper>
+            <GlobusLink uuid={uuid} hubmap_id={hubmap_id} />
+            {visLiftedUUID && <GlobusLink uuid={visLiftedUUID} isSupport />}
+          </DetailSectionPaper>
```

Comment:
Slight preference towards pulling all of this into a component.

Reply:
Done.